.gitignore

```diff
@@ -0,0 +1,2 @@
+redist/
+webpacd.tar.gz
```

Comment:
Add `vscode` or better yet just use one of the standard gitignore templates for the project.

https://github.com/github/gitignore